ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.20__periodics-hcm.yaml

```diff
@@ -0,0 +1,63 @@
+base_images:
+  hypershift-tests:
+    name: hypershift-tests
+    namespace: hypershift
+    tag: latest
+external_images:
+  hypershift-operator:
+    name: rhtap-hypershift-operator
+    namespace: acm-d
+    registry: quay.io
+    tag: latest
+releases:
+  initial:
```

Comment:
Is "initial" used anywhere? 
Otherwise looks good to me.

Reply:
It is.  It is the default when `OCP_IMAGE_PREVIOUS` is not specified.

https://github.com/openshift/release/blob/7a017c92cf5537977569a07f2376a09acbbf7054/ci-operator/step-registry/hypershift/aws/run-e2e/external/hypershift-aws-run-e2e-external-ref.yaml#L18-L19